MVCDemos/Views/FileUploader/CustomDropzone.cshtml

```diff
@@ -74,12 +74,8 @@
 
     function toggleDropZoneActive(dropZone, isActive) {
         if(isActive) {
```

Comment:
We can use .toggle method to add or remove 'dropzone-active' class:
```
dropZone.classList.toggle('dropzone-active', isActive);
```

Reply:
this is the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/8b35563ece420243044d0428445a76dfa8d83aa5)